README.rst

```diff
@@ -791,3 +791,39 @@ For example::
 The ``ZYTE_API_LOG_REQUESTS_TRUNCATE``, 64 by default, determines the maximum
 length of any string value in the logged JSON object, excluding object keys. To
 disable truncation, set it to 0.
+
+
+scrapy-poet integration
+=======================
+
+``scrapy-zyte-api`` includes a `scrapy-poet provider`_ that you can use to get
+data from Zyte API in page objects. Enable it in the Scrapy settings::
+
+    SCRAPY_POET_PROVIDERS = {
+        ZyteApiProvider: 500,
+    }
+
+Request some supported dependencies in the callback::
```

Comment:
The first example shows the dependency in the page object, not in a callback; the second example doesn't show that Product is supported directly in a callback :)